src/api.js

```diff
@@ -5,8 +5,13 @@ const myApi = axios.create({
     baseURL: "https://bmoody-portfolio.onrender.com/api"
 })
 
-export const getReviews = () => {
-    return myApi.get('/reviews').then((res) => {
+export const getReviews = (category) => {
+    let url = `/reviews`
+    if (category) {
```

Comment:
This definitely works! You could also use the axios params here (a second argument passed to the get request to save any if logic)